activator.properties

```diff
@@ -0,0 +1,22 @@
+#
+# Copyright (C) 2015 47 Degrees, LLC http://47deg.com hello@47deg.com
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License. You may obtain
+# a copy of the License at
+#
+#  http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+name=scala-android-preparing-the-environment
+title=Scala in Android - Preparing the environment
+description=A starter application with Scala and Android.
+tags=scala,android
+authorLogo=https://avatars1.githubusercontent.com/u/5580770
```

Comment:
We should use a permalink from our site so we can easily change assets down the road. I tasked @rafaparadela on creating a process for it that will get added to the web README